frontend/awx/resources/inventories/InventoryForm.cy.tsx

```diff
@@ -0,0 +1,344 @@
+import { CyHttpMessages } from 'cypress/types/net-stubbing';
+import { AwxItemsResponse } from '../../common/AwxItemsResponse';
+import { InstanceGroup } from '../../interfaces/InstanceGroup';
+import { Inventory } from '../../interfaces/Inventory';
+import { Label } from '../../interfaces/Label';
+import { Organization } from '../../interfaces/Organization';
+import { CreateInventory, EditInventory, InventoryCreate } from './InventoryForm';
+
+export type regularPayload = {
+  kind: string;
+  name: string;
+  description: string;
+  labels: Array<{ name: string }>;
+  variables: string;
+  prevent_instance_group_fallback: boolean;
+  organization: number;
+};
+
+export type smartPayload = {
+  kind: string;
+  name: string;
+  description: string;
+  variables: string;
+  organization: number;
+  host_filter: string;
+};
+
+describe('Create Edit Inventory Form', () => {
+  const regularPayload = {
```

Comment:
Forgot to use that type above?

const regularPayload : RegularPayload = {....}